src/main/java/programminglife/utility/Alerts.java

```diff
@@ -140,7 +76,7 @@ public void infoAboutAlert() {
     /**
      * Alert method used to show the information from the instructions.
      */
-    public void infoInstructionAlert() {
+    public static void infoInstructionAlert() {
```

Comment:
This long string makes my a bit sad. Maybe it should be included in the resources and loaded from there.